src/main/webapp/app/exercises/programming/shared/lifecycle/programming-exercise-lifecycle.component.html

```diff
@@ -25,7 +25,7 @@
 
             <div class="test-schedule-date px-1">
                 <div class="test-schedule-date-title test-schedule-date-title-small-button">
-                    <span class="fw-bold" jhiTranslate="artemisApp.programmingExercise.timeline.automaticTests"> Automatic Tests </span>
+                    <span jhiTranslate="artemisApp.programmingExercise.timeline.automaticTests"> Automatic Tests </span>
```

Comment:
Fixes this inconsistent bold text: 
![grafik](https://github.com/ls1intum/Artemis/assets/26540346/951bfeac-fa95-40db-8681-2701d8631587)